src/ranch.erl

```diff
@@ -193,24 +193,32 @@ start_error(_, Error) -> Error.
 
 -spec stop_listener(ref()) -> ok | {error, not_found}.
 stop_listener(Ref) ->
+	%% We need to provide an integer timeout to erpc:call,
+	%% otherwise the function will be executed in the calling
+	%% process. 5 minutes should be enough.
+	erpc:call(node(), fun() -> stop_listener1(Ref) end, 300000).
```

Comment:
Also explain why we are using `erpc:call` locally (to avoid reimplementing a spawn/call mechanism).

Reply:
And to avoid the case where the caller process gets killed of course.

Reply:
Done.